openapi/components/schemas/badgeAwardItem.yaml

```diff
@@ -0,0 +1,178 @@
+            description: |
+              The date as a RFC3339 string.
+            example: null
+          goalExpiry:
+            nullable: true
+            type: string
+            format: date-time
+            description: |
+              The date as a RFC3339 string.
+            example: null
+          curriculumNodes:
+            nullable: true
+            type: array
+            items:
+              $ref: ./curriculumItem.yaml
+          tags:
+            nullable: true
+            example: null
+            type: object
+            description: |
+              System configurable in your Admin Lists Learning Moment area
+            properties:
+              id:
+                type: integer
+                description: the DB identifier of the Learning Moment Tags
+                example: 1
+              name:
+                type: string
+                description: the name of the Learning Moment type, may not be unique
+                example: Class Work
+          _links:
+            type: object
+            properties:
+              update:
+                type: string
+                description: |
+                  the link for the user to update the badge
+                example: /learning/badge/10/update
+              delete:
+                type: string
+                description: |
+                  the link for the user to delete the badge
+                example: /learning/badge/10
+              achievementRecipients:
+                type: string
+                example: /learning/badge/award/10/achievement/recipients
+              goalRecipients:
+                type: string
+                example: /learning/badge/award/10/goal/recipients
+              achievementWithdraw:
+                type: string
+                example: /learning/badge/award/10/achievement/withdraw
+              goalWithdraw:
+                type: string
+                example: /learning/badge/award/10/goal/withdraw
+              icon:
+                type: string
+                description: |
+                  The Badge Image, either system defined or selectable by the user
+                example: /static/images/badges/sport/23.svg
+  status:
+    type: string
+    enum:
+      - goal
+      - achievement
+    example: goal
+  isGoal:
+    type: boolean
+    example: true
+  award:
+    $ref: ./badgeAwardItem.yaml
+    nullable: true
+  recipient:
+    $ref: ./userShort.yaml
+  awardedBy:
+    $ref: ./userShort.yaml
+  awardedAt:
+    type: string
+    format: date-time
+    description: |
+      The date as a RFC3339 string.
+    example: 2022-08-27T12:49:11+10:00
+  expiresAt:
+    nullable: true
+    type: string
+    format: date-time
+    description: |
+      The date as a RFC3339 string.
+    example: null
+  viewedOn:
+    nullable: true
+    type: string
+    format: date-time
+    description: |
+      The date as a RFC3339 string.
+    example: null
+  _links:
```

Comment:
Could you add `awardUpdate` here as well?

Reply:
Added